react/src/pages/utils/typings.tsx

```diff
@@ -92,6 +92,8 @@ export interface Analysis {
     started: string;
     finished: string;
     updated_by: number;
+    pipeline?: Pipeline;
+    pipeline_name?: string;
```

Comment:
I see this is because the former is returned by GET /api/analyses/:id and the second now by /api/analyses. Could it be worth it to just join to Pipeline in the list endpoint and return it like in the individual endpoint and leave it up to the frontend to format the display name? Can material-table do that with its render props?

Reply:
I can do that. I think that's how I originally had it in the first commit.